src/components/concept-details-page/index.tsx

```diff
@@ -329,19 +345,20 @@ const ConceptDetailsPage: FC<Props> = ({
                   ))}
                 />
               )}
-              {hiddenLabels.length > 0 && (
-                <KeyValueListItem
-                  property={translations.concept.hiddenLabel}
-                  value={hiddenLabels.map((hiddenLabel, index) => (
-                    <MultiLingualField
-                      key={index}
-                      languages={selectedLanguages}
-                      text={hiddenLabel}
-                      useFallback={false}
-                    />
-                  ))}
-                />
-              )}
+              {hiddenLabels.length > 0 &&
```

Comment:
Det hadde vært fint å bake length sjekk inn i `hasFieldSelectedLanguage` for å slippe gjøre det flere ganger